### ISSUE TITLE
question about the training data of the released model

### ISSUE THREAD

**Issue by LiuJunhong** [google-deepmind/alphafold3#69]
what's the cut-off date of the training data of protein-ligand complexes part of the model that have been released recently? I mean the latest date of the protein-ligand part, because I want to do some evaluation on my tasks and I must figure out that the overlap of my protein-ligand test set and the training ones so that I could construct my test set appropriately. Thanks very much!

**Comment by joshabramson** [comment]
No structural training data past 2021-09-30 was used, as described in the AF3 paper. That covers all molecule types.

**Comment by LiuJunhong** [comment]
thank you very much!
